### PR TITLE
[WIP] Remove lively

### PR DESCRIPTION
**Pull request by eshaffer321**
* Removing the experimental lively library and replacing with regular react hooks.

This is to address this issue:
https://github.com/actualbudget/actual/issues/126

### CODE REVIEW

File: packages/loot-design/src/components/ElementQuery.js [hunk 1]
```diff
@@ -8,30 +13,23 @@ function findMatch(element, sizes) {
...
+    observe(elementRef.current);
+    return () => observe(null);
+  }, []);
 
-export default lively(ElementQuery, {
-  getInitialState() {
-    return { matched: null };
-  },
-  componentDidMount({ inst, props, setState }) {
-    inst.observer = new ResizeObserver(entries => {
```
Comment (eshaffer321) [inline packages/loot-design/src/components/ElementQuery.js:26]: I didn't see a reason why we couldn't use the existing resizeObserver custom hook.

**Comment by eshaffer321** [comment]
This isn't ready for review quite yet. I tried syncing my forked branch with master via rebase and it looks like that didn't achieve what I wanted. I may close with PR and open a new one with my changes versus trying to unwind this.

**Comment by MatissJanis** [comment]
> This isn't ready for review quite yet. I tried syncing my forked branch with master via rebase and it looks like that didn't achieve what I wanted. I may close with PR and open a new one with my changes versus trying to unwind this.

Sure! Whatever is easier for you. :)

**Comment by j-f1** [comment]
Hi @eshaffer321! Would you be able to rebase this PR (also happy to help out with that if you’re having trouble)? I think we should merge this PR asap to avoid future merge conflicts, and in the future try to remove Lively from only one or two components per PR to keep things manageable. Does that make sense?

**Comment by MatissJanis** [comment]
👋 Closing this off for now due the inactivity of 1 month.

Please feel free to open a new PR if you resume working on this. We'd be happy to get rid of lively :)

**Pull request closed without merging**